shipment.go

```diff
@@ -326,47 +323,45 @@ func (c *Client) RerateShipmentWithContext(ctx context.Context, shipmentID strin
 	return
 }
 
+// Deprecated: Use LowestShipmentRate instead.
 // LowestRate gets the lowest rate of a shipment
 func (c *Client) LowestRate(shipment *Shipment) (out Rate, err error) {
-	return c.LowestRateWithCarrier(shipment, nil)
+	return c.LowestShipmentRate(shipment)
 }
 
+// LowestShipmentRate gets the lowest rate of a shipment
+func (c *Client) LowestShipmentRate(shipment *Shipment) (out Rate, err error) {
+	return c.LowestShipmentRateWithCarrier(shipment, nil)
+}
+
+// Deprecated: Use LowestShipmentRateWithCarrier instead.
 // LowestRateWithCarrier performs the same operation as LowestRate,
 // but allows specifying a list of carriers for the lowest rate
 func (c *Client) LowestRateWithCarrier(shipment *Shipment, carriers []string) (out Rate, err error) {
-	return c.LowestRateWithCarrierAndService(shipment, carriers, nil)
+	return c.LowestShipmentRateWithCarrier(shipment, carriers)
 }
 
+// LowestShipmentRateWithCarrier performs the same operation as LowestShipmentRate,
+// but allows specifying a list of carriers for the lowest rate
+func (c *Client) LowestShipmentRateWithCarrier(shipment *Shipment, carriers []string) (out Rate, err error) {
+	return c.LowestShipmentRateWithCarrierAndService(shipment, carriers, nil)
+}
+
+// Deprecated: Use LowestShipmentRateWithCarrierAndService instead.
```

Comment:
Question: Have we confirmed the godoc pics up on this correctly or do we instead need to have this be the 2nd line in the summary?